Main.cs

```diff
@@ -536,17 +538,22 @@ void HandleBlock(Player pl, Message m)
 						if (b >= 93 && b <= 95) edit = true;	// Ice
 						if (b >= 96 && b <= 98) edit = true;	// Gothic
 						if (b >= 100 && b <= 101) edit = true;	// Coins
-						//if (b >= 110 && b <= 111 && pl.isOwner) edit = true;
+						if (b >= 110 && b <= 111) edit = true;  // Secret Coins
```

Comment:
These are shadowed coins and mess up the calculation when placed and removed in a world.
In collected state they have no purporse, so please leave as how it was before.